bundles/org.openhab.binding.pushover/README.md

```diff
@@ -133,6 +133,8 @@ actions.sendMessage("Boost has been activated", "Recuperator", Duration.ofHours(
 var pushoverActions = actions.thingActions('pushover', 'pushover:pushover-account:account');
 // send expiring message
 pushoverActions.sendMessage("Boost has been activated", "Recuperator", time.Duration.ofHours(1));
+// in case you want to send the content of an Image Item (RawType)
+pushoverActions.sendAttachmentMessage("Hello World!", "openHAB", myImageItem.rawState.toFullString(), null);
```

Comment:
Can you please move this to the corresponding DSL example, i.e. after line 102, and make it a full example? Please add the tab markers (`:::: tabs` etc.) to create a new JavaScript tab.

Reply:
I don‘t understand your point. I added this into the existing JavaScript tab, because it is JavaScript syntax. In the above part with DSL Rule there is already a similar example. 
Can you please check this again?

Reply:
There are four different examples, each having their own code fence (e.g. starting with ```java). Only the last example currently has a JavaScript version. This last example shows expiring messages. The example you have provided is about attachment messages, and it looks like a JavaScript version of lines 101-102 (the second example), but you have provided it in the wrong example (number 4).

To fix this you need to add the markers and a JavaScript tab for the second example, and provide the full example (i.e. a JavaScript version of lines 96-102).